tests/test_bs.py

```diff
@@ -39,6 +39,22 @@ def get_elastic_search_api():
     return es
 
 
+def wait_to_deployment_to_be_deleted(deployment_name, name_space, time_out=None):
+    total_sleep_time = 0
+    while True:
+        try:
+            resp = client.AppsV1Api().read_namespaced_deployment(name=deployment_name, namespace=name_space)
+        except ApiException as e:
+            if e.status == 404:
+                print("Total time waiting for delete deployment {0}: {1} sec".format(deployment_name, total_sleep_time))
+                break
+        time.sleep(1)
+        total_sleep_time += 1
+
+        if time_out and total_sleep_time > time_out:
```

Comment:
Can you do X > None where X is an integer?

Reply:
you can't use operator > on different types. (None is not an int)

Reply:
Ok that's why you check it isn't None.